app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,8 @@
+<% provide(:title, "About") %>
+<h1>About</h1>
+<p>
+  The <a href="https://www.railstutorial.org/"> <em>Ruby on Rails Tutorial</em>
+  </a>, part of the <a href="https://www.learnenough.com/">Learn Enough </a>
```

Comment:
I18n và dùng i18n cho toàn bộ các file html,xchú xem lại và fix hét nhé

Reply:
ah em quên I18n,vội quá